apps/site/components/Downloads/ReleaseModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import AlertBox from '@node-core/ui-components/Common/AlertBox';
+import Modal from '@node-core/ui-components/Common/Modal';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import { MinorReleasesTable } from '@/components/Downloads/MinorReleasesTable';
+import LinkWithArrow from '@/components/LinkWithArrow';
+import type { NodeRelease } from '@/types';
+import { getReleaseAnnounceLink } from '@/util/getReleaseAnnounceLink';
+
+type ReleaseModalProps = {
+  isOpen: boolean;
+  closeModal: () => void;
+  release: NodeRelease;
+};
+
+const ReleaseModal: FC<ReleaseModalProps> = ({
+  isOpen,
+  closeModal,
+  release,
+}) => {
+  const t = useTranslations();
+
+  const modalHeadingKey = release.codename
+    ? 'components.releaseModal.title'
+    : 'components.releaseModal.titleWithoutCodename';
+
+  const modalHeading = t(modalHeadingKey, {
+    version: release.major,
+    codename: release.codename ?? '',
+  });
+
+  const releaseAnnounceLink = getReleaseAnnounceLink(release);
+
+  return (
+    <Modal open={isOpen} onOpenChange={closeModal} heading={modalHeading}>
+      {releaseAnnounceLink && (
+        <LinkWithArrow href={releaseAnnounceLink}>
```

Comment:
By using this component, the link becomes i18n, which does not work with the blog, which is not i18n.

Reply:
actually, it's because there is no release announcement for v23 and the release is marked as Maintenance

Reply:
I don't know if v23 should be marked as Maintenance

Reply:
No, v23 is in maintenance